DateTimePickerLibrary/build.gradle

```diff
@@ -16,6 +16,7 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:23.3.0'
-    compile 'com.android.support:appcompat-v7:23.3.0'
+    compile 'com.android.support:support-v4:24.0.0@aar'
+    compile 'com.android.support:appcompat-v7:24.0.0@aar'
```

Comment:
For lean dependencies. You can run gradle task - [androidDependencies] and see those result.

No ```@aar```
```
Information:Gradle tasks [androidDependencies]
:DateTimePickerLibrary:androidDependencies
debug
+--- com.android.support:support-v4:24.0.0
|    \--- LOCAL: internal_impl-24.0.0.jar
\--- com.android.support:appcompat-v7:24.0.0
     +--- com.android.support:support-v4:24.0.0
     |    \--- LOCAL: internal_impl-24.0.0.jar
     +--- com.android.support:support-vector-drawable:24.0.0
     |    \--- com.android.support:support-v4:24.0.0
     |         \--- LOCAL: internal_impl-24.0.0.jar
     \--- com.android.support:animated-vector-drawable:24.0.0
          \--- com.android.support:support-vector-drawable:24.0.0
               \--- com.android.support:support-v4:24.0.0
                    \--- LOCAL: internal_impl-24.0.0.jar
```

With ```@aar```
```
Information:Gradle tasks [androidDependencies]
:DateTimePickerLibrary:androidDependencies
debug
+--- com.android.support:support-v4:24.0.0
|    \--- LOCAL: internal_impl-24.0.0.jar
\--- com.android.support:appcompat-v7:24.0.0
```